tests/core/a-scene.test.js

```diff
@@ -124,3 +121,73 @@ helpers.getSkipCISuite()('a-scene (with renderer)', function () {
     sinon.assert.calledWith(Component.tick, scene.time);
   });
 });
+
+helpers.getSkipCISuite()('a-scene (for iOS)', function () {
```

Comment:
I can help with this in a follow-up patch, but I don't think we need rendering for these tests, and they can move to a `metaTags.test.js` suite.